Source/WebCore/Modules/webauthn/AuthenticationExtensionsClientInputs.cpp

```diff
@@ -50,6 +50,28 @@ std::optional<AuthenticationExtensionsClientInputs> AuthenticationExtensionsClie
     it = decodedMap.find(cbor::CBORValue("credProps"));
     if (it != decodedMap.end() && it->second.isBool())
         clientInputs.credProps = it->second.getBool();
+    it = decodedMap.find(cbor::CBORValue("largeBlob"));
+    if (it != decodedMap.end() && it->second.isMap()) {
+        const auto& largeBlobMap = it->second.getMap();
+        AuthenticationExtensionsClientInputs::LargeBlobInputs largeBlob;
+        auto largeBlobIt = largeBlobMap.find(cbor::CBORValue("support"));
+
+        if (largeBlobIt != largeBlobMap.end() && largeBlobIt->second.isString())
+            largeBlob.support = largeBlobIt->second.getString();
+
+        largeBlobIt = largeBlobMap.find(cbor::CBORValue("read"));
+        if (largeBlobIt != decodedMap.end() && largeBlobIt->second.isBool())
+            largeBlob.read = largeBlobIt->second.getBool();
+
+        largeBlobIt = largeBlobMap.find(cbor::CBORValue("write"));
+        if (largeBlobIt != decodedMap.end() && largeBlobIt->second.isByteString()) {
+            RefPtr<ArrayBuffer> write = ArrayBuffer::create(largeBlobIt->second.getByteString().data(), largeBlobIt->second.getByteString().size());
```

Comment:
I think you probably want to use the helper you defined in ArrayBuffer here. ByteString is a CBOR BinaryValue, which is defined as `using BinaryValue = Vector<uint8_t>;`